src/Sylius/Bundle/ThemeBundle/Twig/ThemeFilesystemLoader.php

```diff
@@ -70,6 +72,25 @@ public function getSource($name)
     /**
      * {@inheritdoc}
      */
+    public function getSourceContext($name)
+    {
+        try {
+            $path = $this->findTemplate($name);
+
+            return new \Twig_Source(file_get_contents($path), $name, $path);
+        } catch (\Exception $exception) {
+            // In Twig 2.0, getSourceContext is part of \Twig_LoaderInterface
+            if ($this->decoratedLoader instanceof \Twig_SourceContextLoaderInterface || method_exists('\\Twig_LoaderInterface', 'getSourceContext')) {
```

Comment:
Shouldn't `method_exists('\\Twig_LoaderInterface', 'getSourceContext')` be `method_exists($this->decoratedLoader, 'getSourceContext')`?